docs/reference/ocm_configfile.md

```diff
@@ -168,6 +168,27 @@ The following configuration types are supported:
 
   At least the given values must be present in the certificate
   to be accepted for a successful signature validation.
+- <code>local.oci.uploader.config.ocm.software</code>
+  The config type <code>local.oci.uploader.config.ocm.software</code> can be used to set some
```

Comment:
This should become part of the generic oci uploader configuration, e.g. `oci.uploader.config.ocm.software` because there is no semantic relationship between a "local.oci.uploader" and the relative access setup. (localhost is only used in the given examples and in tests but it could be any repository).